osu.Game/Graphics/Containers/OsuFocusedOverlayContainer.cs

```diff
@@ -53,7 +53,14 @@ private void load(AudioManager audio)
 
             samplePopIn = audio.Samples.Get(@"UI/overlay-pop-in");
             samplePopOut = audio.Samples.Get(@"UI/overlay-pop-out");
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
 
+            // This must be added after the base LoadComplete. The overlay may need to be hidden immediately if its disabled,
```

Comment:
i cannot understand this comment no matter how many times/ways i read it.

Reply:
Will try again, but basically the `OsuFocusedOverlayContainer` is responsible for blocking the overlay toggle in the case of `OverlayActivationMode.Disabled`, but it can't block it unless the rest of the activation process has already been completed, since it just toggles `Hide` which does nothing if everything happens after the `Hide`.